thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -353,18 +379,32 @@ export default Ember.Controller.extend({
     'hasErrorsBreakdowns'
   ),
 
+  //
+  // session handling
+  //
+  sessionCanSave: Ember.computed(
+    'sessionOwner',
+    'username',
+    function () {
+      const { sessionOwner, username } = this.getProperties('sessionOwner', 'username');
+      return _.isEmpty(sessionOwner) || (sessionOwner === username);
+    }
+  ),
+
+  sessionCanCopy: Ember.computed.bool('sessionId'),
+
   /**
    * Sets the transient rca session properties after saving
    *
    * @param sessionId rca session id
    * @private
    */
   _updateSession(sessionId) {
-    const sessionUpdatedBy = this.get('authService.data.authenticated.name'); // fetch ldap of current user
+    const { username } = this.getProperties('username');
```

Comment:
lol Alex...

Reply:
deja vu. this time we have permissions tracking